error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -639,4 +639,16 @@ Stream<T> after(@Repeated Stream<T> stream) {
       return Streams.concat(Refaster.asVarargs(stream));
     }
   }
+
+  static final class StreamsTakeWhileFilter<T> {
```

Comment:
It's about the `@AfterTemplate` :)
```suggestion
  static final class StreamTakeWhile<T> {
```